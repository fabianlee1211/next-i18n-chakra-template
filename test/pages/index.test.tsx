import React from 'react';
import { render } from '../testUtils';
import Home from '../../pages/index';

jest.mock('next/config', () => () => ({
  publicRuntimeConfig: { en: 'en', 'zh-HK': 'zh-hk' },
}));

describe('Home page', () => {
  it('matches snapshot', async () => {
    const { asFragment } = render(<Home />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
