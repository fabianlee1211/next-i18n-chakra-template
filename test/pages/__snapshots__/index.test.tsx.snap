// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-3z75l1"
  >
    <h2
      class="chakra-heading css-zey6tx"
    >
      Next.js i18n with Chakra UI Template
    </h2>
    <h2
      class="chakra-heading css-jk7tnf"
    >
      Translated Text
    </h2>
    <p
      class="chakra-text css-17t5ffy"
    >
      Example repository of
    </p>
    <div
      class="chakra-stack css-18dncob"
    >
      <code
        class="chakra-code css-1dvx7i9"
      >
        Next.js
      </code>
      <code
        class="chakra-code css-1dvx7i9"
      >
        chakra-ui
      </code>
      <code
        class="chakra-code css-1dvx7i9"
      >
        typescript
      </code>
      <code
        class="chakra-code css-1dvx7i9"
      >
        jest
      </code>
      <code
        class="chakra-code css-1dvx7i9"
      >
        i18n
      </code>
    </div>
    <button
      class="chakra-button css-16pn0c5"
      type="button"
    >
      Change Language
    </button>
    <p
      class="chakra-text css-0"
    >
      Next ❤️ Chakra
    </p>
  </div>
</DocumentFragment>
`;
